tests/lax_scipy_sparse_test.py

```diff
@@ -218,14 +221,16 @@ def test_cg_weak_types(self):
     shape=[(5, 5)],
     dtype=[np.float64, np.complex128],
     preconditioner=[None, 'identity', 'exact', 'random'],
+    symmetric=[False, True]
   )
-  def test_bicgstab_against_scipy(
-      self, shape, dtype, preconditioner):
+  def test_bicgstab_against_scipy(self, shape, dtype, preconditioner, symmetric):
     if not config.enable_x64.value:
       raise unittest.SkipTest("requires x64 mode")
 
     rng = jtu.rand_default(self.rng())
     A = rng(shape, dtype)
+    if symmetric:
+      A = A @ A.T.conj()
```

Comment:
This makes `A` hermitian, but does not necessarily make `A` symmetric. Maybe use `A = A + A.T`?